task_opts.go

```diff
@@ -39,7 +39,7 @@ func WithProcessKill(ctx context.Context, p Process) error {
 	// the process and don't care about the exit status
 	s, err := p.Wait(ctx)
 	if err != nil {
-		return err
+		return nil
```

Comment:
yuck

Reply:
The comment above said to ignore errors of Wait and Kill :angel:

Reply:
well, until now the error did not equal exit status, it was for internal grpc errors and such